examples/quickstart/main.rs

```diff
@@ -439,24 +454,30 @@ async fn get_tx_requests(
         .await
         .expect("Failed to get nonce");
 
-    let approval_request = TransactionRequest::default()
-        .from(user_address)
-        .to(sell_token_address)
-        .input(TransactionInput { input: Some(AlloyBytes::from(data)), data: None })
-        .gas_limit(50_000u64)
-        .max_fee_per_gas(max_fee_per_gas.into())
-        .max_priority_fee_per_gas(max_priority_fee_per_gas.into())
-        .nonce(nonce);
-
-    let swap_request = TransactionRequest::default()
-        .to(Address::from_slice(&tx.to))
-        .from(user_address)
-        .value(biguint_to_u256(&tx.value))
-        .input(TransactionInput { input: Some(AlloyBytes::from(tx.data)), data: None })
-        .max_fee_per_gas(max_fee_per_gas.into())
-        .max_priority_fee_per_gas(max_priority_fee_per_gas.into())
-        .gas_limit(300_000u64)
-        .nonce(nonce + 1);
+    let approval_request = TransactionRequest{
```

Comment:
why did you change having the builder pattern to initing directly? I was forcing myself to use the builder pattern to get more used to it and be more rusty xD

Reply:
The builder pattern didn't allow us to change chain_id. It was hardcoded to 1